bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+    Cancelled = 6
+    Rumored = 7
+
+
+class AgeRatingCategories(IntEnum):
+    """IGDB API Age Rating categories IDs."""
+
+    ESRB = 1
+    PEGI = 2
+
+
+class AgeRatings(IntEnum):
+    """PEGI/ESRB ratings IGDB API IDs."""
+
+    Three = 1
+    Seven = 2
+    Twelve = 3
+    Sixteen = 4
+    Eighteen = 5
+    RP = 6
+    EC = 7
+    E = 8
+    E10 = 9
+    T = 10
+    M = 11
+    AO = 12
+
+
+class Games(Cog):
+    """Games Cog contains commands that collect data from IGDB."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+        # Initialize genres
+        asyncio.get_event_loop().create_task(self._get_genres())
+
+    async def _get_genres(self) -> None:
+        """Create genres variable for games command."""
+        body = "fields name; limit 100;"
+        async with self.http_session.get(BASE_URL + "genres", data=body, headers=HEADERS) as resp:
+            result = await resp.json()
+
+        genres = {genre['name'].capitalize(): genre['id'] for genre in result}
+
+        self.genres = {}
+
+        # Manual check genres, replace sentences with words
+        for genre in genres:
+            if genre == "Role-playing (rpg)":
+                self.genres["Role-playing"] = genres[genre]
+                self.genres["Rpg"] = genres[genre]
+            elif genre == "Turn-based strategy (tbs)":
+                self.genres["Turn-based-strategy"] = genres[genre]
+                self.genres["Tbs"] = genres[genre]
+            elif genre == "Real time strategy (rts)":
+                self.genres["Real-time-strategy"] = genres[genre]
+                self.genres["Rts"] = genres[genre]
+            elif genre == "Hack and slash/beat 'em up":
+                self.genres["Hack-and-slash"] = genres[genre]
+            else:
+                self.genres[genre] = genres[genre]
+
+    @group(name='games', aliases=['game'], invoke_without_command=True)
+    async def games(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
+        """
+        Get random game(s) by genre from IGDB. Use .movies genres command to get all available genres.
+
+        Also support amount parameter, what max is 25 and min 1, default 5. Use quotes ("") for genres with multiple
+        words.
+        """
+        # Capitalize genre for check
+        genre = genre.capitalize()
+
+        # Check for amounts, max is 25 and min 1
+        if amount > 25:
+            await ctx.send("You can't get more than 25 games at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 game.")
+            return
+
+        # Get games listing, if genre don't exist, show help.
+        try:
+            games = await self.get_games_list(self.http_session, amount, self.genres[genre],
+                                              offset=random.randint(0, 150))
+        except KeyError:
+            await ctx.send_help('games')
```

Comment:
I don't think just sending the command's help here is useful. There should be feedback to the user that there's something wrong with the genre they provided.